From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@aniedea__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (aniedea). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

aniedea commented:
typo -- select "SPARK" not "Hadoop" for cluster type

azurecla commented:
Hi __@aniedea__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (aniedea). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@nitinme 
fyi

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Add files via upload

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-sso-how-it-works.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Add files via upload

swkrish committed:
Add files via upload

swkrish committed:
Add files via upload

swkrish committed:
Delete sso13.png

swkrish committed:
Add files via upload

swkrish committed:
Add files via upload

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Add files via upload

swkrish committed:
Add files via upload

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Add files via upload

swkrish committed:
Add files via upload

swkrish committed:
Delete sso14.png

swkrish committed:
Add files via upload

swkrish committed:
Merge branch 'master' into master

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

v-nagta committed:
USER STORY 1190836 SaaS App Tutorial: Andromeda SCM Tutorial

v-nagta committed:
updated

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-smart-lockout.md

Commit:
Upload file using azcopy

Specifies whether the destination blob is a block blob, a page blob, or an append blob. This option is applicable only when you are uploading a blob. Otherwise, an error is generated. If the destination is a blob and this option is not specified, by default, AzCopy creates a block blob.

v-nagta committed:
Appilication name changed to andromeda

billmath committed:
pulling in PIM updates

billmath committed:
adding updates

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into issues

tamram committed:
adding SSE update for lakasa

tamram committed:
adding SSE update for lakasa

tamram committed:
adding SSE update for lakasa

tamram committed:
adding SSE update for lakasa

tamram committed:
adding SSE update for lakasa

tamram committed:
adding SSE update for lakasa

v-nagta committed:
USER STORY 1200762 SaaS App Tutorial: People Tutorial Update

v-nagta committed:
updated

tamram committed:
updates for SSE by default

tamram committed:
updating docs for SSE by default

tamram committed:
metadata and acrolinx

tamram committed:
one more

tamram committed:
updates for customer-managed keys

tamram committed:
fixing key vault refs

tamram committed:
updated images

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0305

tamram committed:
update images

tamram committed:
update images

tamram committed:
update images

tamram committed:
fix step 3

tamram committed:
fixing H2s

tamram committed:
fixing indent

v-nagta committed:
url and attribute section

v-nagta committed:
url section updated

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
updated

v-nagta committed:
missing icons add

CarlRabeler committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into MI

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-current-limitations.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-quick-start.md

swkrish committed:
Update active-directory-aadconnect-pass-through-authentication-smart-lockout.md

swkrish committed:
Update active-directory-aadconnect-sso-faq.md

swkrish committed:
Update active-directory-aadconnect-sso.md

swkrish committed:
Update active-directory-aadconnect-troubleshoot-sso.md

CarlRabeler committed:
tweaks

Commit:
added the port diagram

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0306

Commit:
added port requirements table

Commit:
renamed PNG

CarlRabeler committed:
more

tamram committed:
updating image for portal

tynevi committed:
Update CODEOWNERS

CarlRabeler committed:
acrolinx

CarlRabeler committed:
Merge branch 'master' into MI

CarlRabeler committed:
bad link

tynevi committed:
Update CODEOWNERS

Commit:
added ts for vCenter

Commit:
removing extra code blocks

Commit:
added link to upgrade

tynevi committed:
Update CODEOWNERS

Commit:
Merge https://github.com/cynthn/azure-docs-pr into 5205

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into issues

billmath committed:
fixing per review and tyson feedback

PRMerger16 committed:
Merge pull request #34826 from cynthn/5205

Edits from public PR 5205

v-shils committed:
Merge pull request #34804 from ruturaj/master

Collector ports

rmca14 committed:
Merge pull request #34533 from billmath/issues

Issues

rmca14 committed:
Merge pull request #32702 from swkrish/master

Feb updates to PTA & Seamless SSO

CarlRabeler committed:
en-us

CarlRabeler committed:
en-us

CarlRabeler committed:
Merge branch 'MI' of https://github.com/CarlRabeler/azure-docs-pr into MI

CarlRabeler committed:
en-us/

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #34602 from tamram/tamram-0305

adding SSE update for lakasa

Ja-Dunn committed:
Merge pull request #34777 from v-nagta/misicon

missing icons add

Ja-Dunn committed:
Merge pull request #34807 from CarlRabeler/MI

Mi

rjagiewich committed:
Merge pull request #33995 from v-nagta/andromeda

USER STORY 1190836 SaaS App Tutorial: Andromeda SCM Tutorial

v-alje committed:
Merge pull request #34840 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #34624 from v-nagta/people

USER STORY 1200762 SaaS App Tutorial: People Tutorial Update

Lisaco88 committed:
Merge pull request #34695 from tamram/tamram-0306

updates for SSE by default

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
